docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -41,6 +42,17 @@ Other than those considerations listed here, SOA Suite domains work in the same
 In this release, SOA Suite domains are supported using the “domain on a persistent volume”
 [model]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) only, where the domain home is located in a persistent volume (PV).
 
+#### Prerequisites for SOA Suite Domains
+
+* Kubernetes 1.11.5+, 1.12.3+, 1.13.0+, 1.14.0+, and 1.15.0+ (check with `kubectl version`).
```

Comment:
have you actually tested all of these?
i would *strongly* recommend that you limit to 1.13-1.15 and not go all the way back to 1.11
there are sec vuls on some of these too, so you cant go with the .0's - you need to check the cve's and pick dot releases that are safe

Reply:
Hi Mark, 

Thanks for the feedback, yes we should limit these to 1.13.0+ to 1.15.0+. Will also try to get the not dot release that has been tested. We will fix this. However, we pulled in these data from the FMW Infrastructure domain pre-req. Here is the link : https://oracle.github.io/weblogic-kubernetes-operator/userguide/introduction/introduction/

I think you should fix that too to remain consistent.

Reply:
thanks @rdas76 - will do
@rjeberhard fyi -- our fmw infra list is using the .0's - we should fix that (or just point back to the main list)